src/components/ux-tooltip/ux-tooltip.feature

```diff
@@ -0,0 +1,8 @@
+Feature: ux-tooltip test suite
+
+	Scenario: Loading ux-tooltip
+		Given I have loaded component "ux-tooltip" with use case "dataDriven"
+		When I hover "link"
```

Comment:
Created a new issue for this: https://github.com/ractive-foundation/ractive-foundation/issues/255